### PR TITLE
Updating Postgres extended query protocol section

### PR DESCRIPTION
**Pull request by sadhbh-a**
### What does this PR do? What is the motivation?
The query_samples parameter is no longer in beta, and is actually enabled by default for Postgres 12 & above.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by aldrickdev — APPROVED** [review]
👍 thanks for the update.

File: content/en/database_monitoring/setup_postgres/troubleshooting.md [hunk 1]
```diff
@@ -188,14 +188,14 @@ See the section on [truncated query samples](#query-samples-are-truncated) for i
 
 If a client is using the Postgres [extended query protocol][9] or prepared statements, the Datadog Agent is unable to collect explain plans due to the separation of the parsed query and raw bind parameters. Below are a few options for addressing the problem.
 
-For Postgres version 12 or newer, enable the following beta feature in the [Postgres integration config][19].
+For Postgres version 12 or newer, the below parameter is enabled by default in the [Postgres integration config][19], allowing the Agent to collect explain plans:
```
Comment (neko-dd) [inline content/en/database_monitoring/setup_postgres/troubleshooting.md:191]:
```suggestion
For Postgres version 12 or newer, the following parameter is enabled by default in the [Postgres integration config][19], allowing the Agent to collect explain plans:
```

**Review by neko-dd — APPROVED** [review]
Same deal - one suggestion but otherwise LGTM :)

**Pull request merged**